internal/ingress/status/status.go

```diff
@@ -226,8 +226,21 @@ func (s *statusSync) runningAddresses() ([]apiv1.LoadBalancerIngress, error) {
 }
 
 func (s *statusSync) isRunningMultiplePods() bool {
+
+	// As a standard, app.kubernetes.io are "reserved well-known" labels.
+	// In our case, we add those labels as identifiers of the Ingress
+	// deployment in this namespace, so we can select it as a set of Ingress instances.
+	// As those labels are also generated as part of a HELM deployment, we can be "safe" they
+	// cover 95% of the cases
+	podLabel := make(map[string]string)
+	for k, v := range k8s.IngressPodDetails.Labels {
+		if k != "pod-template-hash" {
```

Comment:
The logic here can only be processed for `Deployment`  when user using `DaemonSet` it will not include this label.
It using `controller-revision-hash` and `pod-template-generation` label.